my-app/src/pages/Delivery/components/RouteReportsPreview.css

```
@@ -0,0 +1,355 @@
+
+.route-report-deliveries {
+  display: grid;
+  gap: 14px;
+}
+
+.route-report-delivery-card {
+  padding: 15px 16px;
+  border: 1.5px solid #9aa5a1;
+  border-left: 5px solid #257e68;
+  break-inside: avoid;
+  page-break-inside: avoid;
```

Stylelint (17.14.0)

[error] Expected "page-break-inside" to be "break-inside" (property-no-deprecated)

(property-no-deprecated)